www/api/search

```diff
@@ -66,11 +67,240 @@ soon as they're available in the UI.
 
 <p>To search for a game listing for <tt>Deep Space Drifter</tt>:
 
-<p><pre>https://ifdb.org/search?xml&amp;game&amp;searchfor=Deep+Space+Drifter
+<p><pre>https://ifdb.org/search?json&amp;game&amp;searchfor=Deep+Space+Drifter
 </pre>
 
 <h2>Reply</h2>
 
+The response is a JSON object, like this:
+
+<p><pre>
+{
+  "games": [
+    {
+      "tuid": "k82q3libhff6ks8l",
+      "title": "Deep Space Drifter",
+      "link": "https://ifdb.org/viewgame?id=k82q3libhff6ks8l",
+      "author": "Michael J. Roberts and Steve McAdams",
+      "hasCoverArt": true,
+      "devsys": "TADS 2",
+      "published": {
+        "machine": "1990",
+        "printable": "1990"
+      },
+      "averageRating": 2.428571428,
+      "numRatings": 7,
+      "starRating": 2.5,
+      "starSort": 2.16074024927881,
+      "coverArtLink": "https://ifdb.org/coverart?id=k82q3libhff6ks8l&version=4"
+    }
+  ]
+}
+</pre></p>
+
+<p>A search that yields no results will simply return an empty "group"
+tag.  (A search with zero results isn't an error; it's a search that
+successfully determined that there were no items in the database
+matching the search terms.)
+
+<p>The contents of the item tags vary according to the object type.  The
+following sections describe the results for the different object types.
+
+<p>Most of the item replies include a "TUID" value ("tads.org Unique
```

Comment:
Link to https://ifdb.org/help-tuid

Reply:
Nahhh, basically all of the words of `help-tuid` are already included inline here in the documentation.

Reply:
It's odd to have all this "what is the thinking behind the TUID design" here. If you want to keep it documented somewhere, maybe it belongs in that "what's a TUID" page.